### PR TITLE
Fix up packaging of wapm on Windows

### LINKED ISSUE #1988: WAPM is missing in installer for Windows
### Describe the bug
Written in the title.

`wasmer 1.0.0`
`rustc 1.48.0 (7eac88abb 2020-11-16)`

### Steps to reproduce
1. Read https://docs.wasmer.io/ecosystem/wapm/getting-started, see `WAPM comes bundled with Wasmer`
1. Go to https://github.com/wasmerio/wasmer/releases/tag/1.0.0
2. Download exe and install wasmer, or install wasmer with PowerShell `iwr https://win.wasmer.io -useb | iex`
2. Open cmd.exe, run `wasmer -V`, result: `wasmer 1.0.0`
3. Run `wapm`, see `'wapm' is not recognized as an internal or external command, operable program or batch file.` or in PowerShell `wapm : The term 'wapm' is not recognized as the name of a cmdlet, function, script file, or operable program.`
4. ...
5. Goto wapm repo https://github.com/wasmerio/wapm-cli and compile it from sources :(.
### Expected behavior
Wapm install with wasmer from wasmer.exe
### Additional context
Windows 10 Pro.

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
It appears to work on Windows

resolves #1988, resolves half of #1971

# Review

- [x] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -286,12 +286,16 @@ test-integration:
 #############
 
 package-wapm:
-ifneq ($(OS), Windows_NT)
 	mkdir -p "package/bin"
+ifneq ($(OS), Windows_NT)
+	if [ -d "wapm-cli" ]; then \
+		cp wapm-cli/target/release/wapm package/bin/ ;\
```
Comment (syrusakbary) [inline Makefile:292]: I think we need to move this up too

Comment (syrusakbary) [reply Makefile:292]: Oh, got confused!

Comment (MarkMcCaskey) [reply Makefile:292]: I tried to but I couldn't get the `ifneq` working internally because the outer loop was bash, so I settled to just duplicate it!

**Review by syrusakbary — APPROVED** [review]

**Pull request merged**